docs/Documentation/Applications/Matlab/interactive.md

```diff
@@ -10,11 +10,10 @@ Connection](https://www.nrel.gov/hpc/system-connection.html).
 
 ## Running MATLAB via an Interactive Job
 
-After connecting to the login node, the next step is to start an interactive job. For
-example, the following command gets # nodes for interactive use for an hour:
+After connecting to the login node, the next step is to start an interactive job. For example, the following command gets a user-selected number of nodes for interactive use, taking as input tasks per node, job duration, account, and partition.
 
 ```
-$ srun -N # --time 01:00:00 --account=<handle> --pty $SHELL
+$ salloc --nodes=<number of nodes> --ntasks-per-node=<tasks per node> --account=<your account here> --time=<desired time> --partition=<desired partition>
```

Comment:
We're still recommending that users don't specify a partition in their job scripts unless using debug. I think the partition flag could be removed.

Reply:
Sae as above: the partition flag can be removed.

Reply:
I removed the partition specification.